spec/lib/client/middleware_spec.rb

```diff
@@ -71,10 +95,10 @@ def perform(_)
 
     it 'enqueues previously scheduled job' do
       QueueWorker.sidekiq_options unique: true
-      jid = QueueWorker.perform_in(60 * 60, 1, 2)
+      QueueWorker.perform_in(60 * 60, [1, 2])
```

Comment:
I think this test is still broken and isn't testing for a job moving from the scheduled to its normal queue. 

Right now this line is creating a scheduled job with a single argument of a two element array and the later `Sidekiq::Client.push` is creating a job with two Fixnum arguments. It's not the same job and the test is only showing that two separate jobs can be created, not that the same job can move from the scheduled to a normal queue. 

Changing this line to `QueueWorker.perform_in(60 * 60, 1, 2)` should be enough to demonstrate the issue